tests/ui/traits/new-solver/provisional-result-done.rs

```diff
@@ -0,0 +1,38 @@
+// known-bug: unknown
```

Comment:
currently still ICEs because other part of the solver aren't implemented yet :sweat_smile: